src/monosynth.js

```diff
@@ -0,0 +1,358 @@
+    var options = this[preset];
+    this.oscillator.setType(options.oscillator.type);
+
+    this.env.setADSR(options.env.attack, options.env.decay,
+            options.env.sustain, options.env.release);
+
+    this.filter.setType(options.filter.type);
+    this.filter.set(options.filter.freq, options.filter.res);
+    return this;
+  };
+
+  //PRESETS
+  p5.MonoSynth.prototype.default = {
+    'oscillator' : {
+      'type' : 'sine'
+    },
+    'env' : {
+      'attack' : 0.02,
+      'decay' : 0.25,
+      'sustain': 0.05,
+      'release': 0.35
+    },
+    'filter': {
+      'type' : 'highpass',
+      'freq' : 5,
+      'res' : 1
+    }
+  };
+
+  p5.MonoSynth.prototype.simpleBass = {
+    'oscillator' : {
+      'type' : 'square'
+    },
+    'env' : {
+      'attack': 0,
+      'decay': .60,
+      'sustain': 0.1,
+      'release': .28
+    },
+    'filter' : {
+      'type' : 'lowpass',
+      'freq' : 15000,
+      'res' : 1
+    }
+  };
+
+  p5.MonoSynth.prototype.electricPiano = {
+    'oscillator' : {
+      'type' : 'sine'
+    },
+    'env' : {
+      'attack': 0.029,
+      'decay': .16,
+      'sustain': 0.1,
+      'release': .1
+    },
+    'filter': {
+      'type' : 'lowpass',
+      'freq' : 15000,
+      'res' : 1
+    }
+  };
+
+  /**
+     *  Set values like a traditional
+     *  <a href="https://en.wikipedia.org/wiki/Synthesizer#/media/File:ADSR_parameter.svg">
+     *  ADSR envelope
+     *  </a>.
+     *  
+     *  @method  setADSR
+     *  @param {Number} attackTime    Time (in seconds before envelope
+     *                                reaches Attack Level
+     *  @param {Number} [decayTime]    Time (in seconds) before envelope
+     *                                reaches Decay/Sustain Level
+     *  @param {Number} [susRatio]    Ratio between attackLevel and releaseLevel, on a scale from 0 to 1,
+     *                                where 1.0 = attackLevel, 0.0 = releaseLevel.
+     *                                The susRatio determines the decayLevel and the level at which the
+     *                                sustain portion of the envelope will sustain.
+     *                                For example, if attackLevel is 0.4, releaseLevel is 0,
+     *                                and susAmt is 0.5, the decayLevel would be 0.2. If attackLevel is
+     *                                increased to 1.0 (using <code>setRange</code>),
+     *                                then decayLevel would increase proportionally, to become 0.5.
+     *  @param {Number} [releaseTime]   Time in seconds from now (defaults to 0)
+     **/
+
+  p5.MonoSynth.prototype.setADSR = function (attack,decay,sustain,release) {
+    this.env.setADSR(attack, decay,  sustain, release);
+  };
+
+
+  /**
+   * Getters and Setters
+   * @param {Number} attack
+   * @param {Number} decay
+   * @param {Number} sustain
+   * @param {Number} release
+   */
+  Object.defineProperties(p5.MonoSynth, {
```

Comment:
define these on the `prototype`, so that they're properties of monosynth instances